python/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -149,6 +149,8 @@ def prepareCommand(app):
         full_cmd = sourceEnv + "./run gaudirun.py %s %s" % (' '.join(opts_names), GaudiExecDiracRTHandler.data_file)
         if app.extraOpts:
             full_cmd += ' ' + app.getExtraOptsFileName()
+        if app.getMetadata:
```

Comment:
Actually I think this is best here - I wanted to keep this stuff out of the extraOpts part of the application.